addons/website_hr_recruitment/i18n/website_hr_recruitment.pot

```diff
@@ -790,11 +800,6 @@ msgstr ""
 msgid "for job opportunities."
 msgstr ""
 
-#. module: website_hr_recruitment
-#: model_terms:ir.ui.view,arch_db:website_hr_recruitment.index
-msgid "open positions"
```

Comment:
Are you sure you want to change the punctuation for this? Doesn't seem necessary + it will require it to be re-translated in stable